gpii/node_modules/flowManager/src/Capture.js

```diff
@@ -0,0 +1,229 @@
+/**
+ * GPII snapshot Handler
+ *
+ * Copyright 2014 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/gpii/universal/LICENSE.txt
+ */
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.defaults("gpii.flowManager.capture", {
+    gradeNames: ["fluid.component"],
+    events: {
+        // Pseudoevents for transforming promise chains to fetch the solutions on the current device
+        // and capture the settings for the device.
+        onSolutionsForCurrentDevice: null,
+        onCaptureSettingsForCurrentDevice: null
+    },
+    listeners: {
+        // Begin declaration of Promise Chain for onSolutionsForCurrentDevice
+        "onSolutionsForCurrentDevice.getDeviceContextPromise": {
```

Comment:
It would be clearer to add `priority: "first"` to indicate it's the first listener executed in the promise chain although this is implicitly defaulted to.

The same comment applies to the next promise chain at line 42.

Reply:
added both of these priorities